tsdb/compact.go

```diff
@@ -330,7 +330,10 @@ func splitByRange(ds []dirMeta, tr int64) [][]dirMeta {
 	return splitDirs
 }
 
-func compactBlockMetas(uid ulid.ULID, blocks ...*BlockMeta) *BlockMeta {
+// CompactBlockMetas merges many block metas into one, combining it's source blocks together
+// and adjusting compaction level.
+// NOTE(bwplotka): Used by Thanos code.
```

Comment:
I don't think we need this note

Reply:
Yea, wonder if it's not useful to not get this removed in next commit, happy to remove (:

Reply:
We expose many API's for many third parties;  I don't think we should mention it. I don't think we ever did.

Reply:
I agree. If we need this note, then that indicates this is dead code we should be deleting.